community/cypher/cypher/src/test/scala/org/neo4j/cypher/IndexUsageAcceptanceTest.scala

```diff
@@ -125,6 +125,30 @@ class IndexUsageAcceptanceTest extends ExecutionEngineFunSuite with NewPlannerTe
     result.toList should equal(List(Map("p" -> null, "placeName" -> null)))
   }
 
+  test("should not use indexes when RHS of property comparison depends on the node searched for (equality)") {
```

Comment:
The test says 'should not use indexes' but it makes no such assertion. I'd say we add one or change the test name.